docs/production-deployment.rst

```diff
@@ -556,3 +556,36 @@ Keytab secret and both containers in the same Pod share the volume, where tempor
 the side-care container and read by the worker container.
 
 This concept is implemented in the development version of the Helm Chart that is part of Airflow source code.
+
+Tips and Tricks for Oracle Shops
```

Comment:
Is there a value to have this paragraph here specifically for Oracle?

We already have howto guide for creating Oracle connection: https://github.com/apache/airflow/blob/master/docs/howto/connection/oracle.rst
Maybe you can enhance it ?